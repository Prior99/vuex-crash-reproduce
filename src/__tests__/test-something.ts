import ParentComponent from "../parent-component.vue";
import { createLocalVue, mount } from "@vue/test-utils";
import Vue from "vue";
import Vuex from "vuex";

test("a test", () => {
    const localVue = createLocalVue();
    localVue.use(Vuex);
    const storeMock = {
        state: { content: "" },
        namespaced: true,
        getters: {
            getContent: jest.fn()
        },
    };

    const store = new Vuex.Store({
        modules: {
            myModule: storeMock
        }
    });

    const wrapper = mount(ParentComponent, { localVue, store });
    wrapper.find("a").trigger("click");
    expect(wrapper.element).toMatchSnapshot();
});
