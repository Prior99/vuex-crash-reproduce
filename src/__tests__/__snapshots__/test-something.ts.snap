// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`a test 1`] = `
<div>
  <p>
    
  </p>
   
  <p>
    original value
  </p>
   
  <button />
</div>
`;
